DeepSpeech.ipynb

```diff
@@ -446,7 +446,7 @@
     "    lstm_bw_cell = tf.nn.rnn_cell.BasicLSTMCell(n_cell_dim, forget_bias=1.0)\n",
     "    \n",
     "    # Split data because rnn cell needs a list of inputs for the BRNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
```

Comment:
I'm not convinced these are equivalent. Imagine that layer_3 had shape (10, 1000) and n_steps was 5, and also imagine that split returned a tensor with an extra dimension instead of a python list. After the split, layer_3 would have shape (2, 5, 1000), but after the unpack, I think, it would have shape (10, 1, 1000)? I'll confirm this when I get to my main machine.